index.html

```diff
@@ -1537,6 +1537,33 @@ <h2 id="text110" class="style3">Websites</h2>
 								</table>
 							</div>
 						</div>
+						<p class="style2">Research</p>
+						<div class="style1 table-wrapper">
+						  <div class="table-inner">
+						    <table>
+						      <thead>
+						        <tr>
+						          <th>Class</th>
+						          <th>Description</th>
+						        </tr>
+						      </thead>
+						      <tbody>
+						        <tr>
+						          <td><a href="https://github.com/6102bitcoin/CoinJoin-Research">Coinjoing Research</a></td>
```

Comment:
"Coinjoing" needs spelling correction.

Reply:
Fixed! Wow github is pretty fucking cool, didn't know it would auto update it.